src/Session/commands.py

```diff
@@ -1,14 +1,48 @@
 from src.Constants.commands_messages import *
 from src.Constants.constants import *
+from src.Food.food import Food
 from src.Session.profile import Profile
 from src.Utils.utils import listInQuotes, twoPrintOutcomes
 
 class Commands:
     def __init__(self):
         self.profile = None
 
+    ''' Prompts user to add foods to whichever food storage they are on. If the user is not in an opened 
+        food storage, suggests the user opens an existing food storage or create a new one. '''
     def add_food(self):
-        pass
+        # Profile check
+        if not self.profile:
+            print(NO_LOADED_PROFILE_MESSAGE, \
+                SUGGESTED_ACTIONS_MESSAGE.format(listInQuotes([CREATE_PROFILE, LOAD])))
+            return
+        
+        # Check if food storage is open
+        if self.profile.getOpenFoodStorage() != None:
+            food_list = []
+            while True: 
+                food_name = input(ADD_FOOD_NAME_MESSAGE)
+                expiration_date = input(ADD_FOOD_EXPIRATION_DATE_MESSAGE)
+                try:
+                    use_by_date = int(input(ADD_FOOD_USE_BY_DATE_MESSAGE))
+                    food = Food(food_name, expiration_date, use_by_date)
+                except:
+                    food = Food(food_name, expiration_date)
+                food_list.append(food)
+                    
+                while True:
+                    add_more_food = input(ADD_MORE_FOOD_MESSAGE)
+                    if add_more_food == YES:
+                        break
+                    elif add_more_food == NO:
+                        self.profile.addFoods(food_list)
+                        print(ADD_FOOD_SUCCESS_MESSAGE_FINAL)
+                        return
+                    else:
+                        print(INVALID_RESPONSE_MESSAGE)
```

Comment:
More unique invalid response messages instead of one generic one could be better also, like for example here you could include in the response message that the valid responses are 'YES' or 'NO'.

Reply:
We reuse this for other similar code. I feel like it kind of already mentions the valid responses when it loops back around. When it prints the invalid response, it loops back and asks the `ADD_MORE_FOOD_MESSAGE` which says `(y/n)` in it.